### PR TITLE
Add VideLibri

### PR DESCRIPTION
**Pull request by benibela**

### CODE REVIEW

**Comment by stweil** [comment]
This is neither an organization or library institution nor a person related to libraries, but a software product for libraries. So it does not fit in the current categories. I am not sure whether we should add a new category for library related software. I am afraid that such a category could grow very fast and hide the initial purpose of BibsOnGitHub, namely listing persons and institutions.

**Comment by benibela** [comment]
I had assumed "projects" in the second caption of the list meant all kind of library related projects, does it not? There is already another software development project, finc, in the list

File: list.md [hunk 1]
```diff
@@ -173,6 +173,7 @@ Zentrale Hochschulbibliothek Lübeck
 Universitätsbibliothek Bamberg 
 [https://github.com/uniba-ub](https://github.com/uniba-ub)
 
+Bibliothek-App VideLibri [https://github.com/benibela/videlibri](https://github.com/benibela/videlibri)
```
Comment (stweil) [inline list.md:176]:
```suggestion
```

File: list.md [hunk 2]
```diff
@@ -380,3 +382,4 @@ David Zellhöfer (SBB Berlin)
 
 Philipp Zumstein (UB Mannheim)
 [https://github.com/zuphilip](https://github.com/zuphilip)
+
```
Comment (stweil) [inline list.md:385]:
```suggestion
```

File: list.md [hunk 3]
```diff
@@ -348,6 +348,8 @@ Fabian Steeg (hbz)
 Marco Tullney (TIB/UB Hannover)
 [https://github.com/tullney](https://github.com/tullney)
 
+Benito van der Zander (VideLibri) [https://github.com/benibela](https://github.com/benibela)
```
Comment (stweil) [inline list.md:351]:
```suggestion
Benito van der Zander (VideLibri)
[https://github.com/benibela](https://github.com/benibela)
```

**Comment by stweil** [comment]
@benibela, I now committed your pull request with only your personal entry which fits to the list and hope that's fine for your, too.

**Pull request merged**